### ISSUE TITLE
Do not force the log level to DEBUG

### ISSUE THREAD

**Issue by nicolas-doby** [Azure/azure-data-lake-store-python#274]
### Description
**Outline the issue here:**
When modifying the ACL entries on a path the following logs are displayed:
```
Sending exception sentinel
Exception monitor thread finished
Sending logger sentinel
Log queue closed
Log thread finished
...
```

Debug logs should only be displayed on demand; not forced.

---

### Reproduction Steps
**Enumerate the steps to reproduce the issue here:**
Call this method:
`auth.adls_file_system_client.modify_acl_entries(path, acl_spec, recursive=True)`

In multuprocessor.py, the log level is forced to DEBUG:
Line 24, 68 and 160: `logger.setLevel(logging.DEBUG)`.

### Environment summary

**SDK Version:** What version of the SDK  are you using? (pip show azure-datalake-store)
Name: azure-datalake-store
Version: 0.0.41
Summary: Azure Data Lake Store Filesystem Client Library for Python
Home-page: https://github.com/Azure/azure-data-lake-store-python
Author: Microsoft Corporation
Author-email: ptvshelp@microsoft.com
License: MIT License
Location: c:\users\nicolas_doby\appdata\local\programs\python\python37-32\lib\site-packages
Requires: adal, cffi, requests
Required-by: azure

**Python Version:** What Python version are you using? Is it 64-bit or 32-bit?  
Python 3.7.0 (64-bit)

**OS Version:** What OS and version are you using?  
Windows 10

**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, Bash on Windows)  
Ubuntu Bash

**Comment by akharit** [comment]
Agreed. Will fix it and release a new version

**Comment by akharit** [comment]
Fixed in 0.0.42
